setup.py

```diff
@@ -2,7 +2,7 @@
 
 setup(
     name='commcare-analytics',
-    version='1.1.2',
+    version='1.2.0',
```

Comment:
I assume we did 1.2.0 and not 1.1.3 because this is not a bug fix and a feature addition which warrants the minor version to be updated?

Reply:
Correct.